bentoml/yatai/python_api.py

```diff
@@ -149,3 +162,174 @@ def update_bento_upload_progress(
         service_metadata=bento_service._get_bento_service_metadata_pb(),
     )
     yatai.UpdateBento(update_bento_req)
+
+
+def create_deployment(
```

Comment:
discussed offline, add yatai_service=None parameter to all related yatai python_api functions